installer/Linux/installer/Readme.md

```diff
@@ -0,0 +1,2 @@
+# Protocol Handler for `ide://...` scheme.
```

Comment:
Readme.md -> README.adoc to be consistent.

Reply:
Done @Commit https://github.com/maxandersen/starfish/pull/19/commits/61c24b5c5dcc867e4112e1c0fa7376295c4ab0cc